.github/workflows/test_openvino_basic.yml

```diff
@@ -22,13 +22,27 @@ jobs:
       matrix:
         # Testing lower and upper bound of supported Python versions
         # This also ensures that the test fails if dependencies break for Python 3.7
-        python-version: ["3.8", "3.12"]
+        python-version: ["3.9", "3.12"]
         os: ["ubuntu-22.04", "windows-latest"]
-        transformers-version: ["4.44.*"]
+        transformers-version: ["4.45.*"]
+        openvino: ["openvino"]
+        nncf: ["nncf"]
         include:
           - python-version: "3.12"
             os: "ubuntu-22.04"
             transformers-version: "4.36.0"
+            openvino: "openvino"
+            nncf: "nncf"
+          - python-version: "3.12"
+            os: "ubuntu-22.04"
+            transformers-version: "4.45.*"
+            openvino: "--pre -U openvino --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly"
```

Comment:
do we need somehow add openvino tokenizers version too? (they may be incompatible if we have openvino nightly and release toknizers, as the result tokenizers related tests will always fail)

Reply:
we currently don't install `openvino-tokenizers` but can be added

Reply:
added in https://github.com/huggingface/optimum-intel/pull/922/commits/4bb97dd1d8c47f1fa270d42825675e639d36ff1e